### PR TITLE
feat(cloudfront constructs): add s3 access logging to cloudfront access log buckets by default

### PR DESCRIPTION
**Pull request by biffgaut**
*Issue #, if available:*
Addresses Issue #1041 

*Description of changes:*
Adds s3 access logging to the CloudFront log buckets by default

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: 3a2043651bf467939b57f7781ccadf42574adc89
* Result: FAILED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=3e19e199-1c7d-46c2-a2fa-27c38e275e64%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: 7a33919a6f9c1c3fc906f6c0184e87ca81fdf3d2
* Result: SUCCEEDED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=a86d4157-4a5d-4837-a439-0041a990841a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: deployment/v2/refresh-multiple-tests.sh [hunk 1]
```diff
@@ -19,82 +19,14 @@
 #   a sleep 10 command before the end of the loop to keep from overwhelming CloudFormation
 
 export constructs="
-  aws-alb-fargate
```
Comment (georgebearden) [inline deployment/v2/refresh-multiple-tests.sh:22]: was removing these intentional?

**Review by georgebearden — APPROVED** [review]
Looks good to me, just one question on a change being intentional

**Review by georgebearden — APPROVED** [review]

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: e59a01bf31578168ead8da0a1e6207060cb7f47d
* Result: SUCCEEDED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=8a6ed19a-2a45-443e-90f9-bb4c53cee2ff%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**